app/src/main/java/org/permanent/permanent/repositories/IProfileRepository.kt

```diff
@@ -0,0 +1,8 @@
+package org.permanent.permanent.repositories
+
+import org.permanent.permanent.network.IDataListener
+
+interface IProfileRepository {
+
+    fun getProfileItemsByArchive(listener: IDataListener)
```

Comment:
I see here the method is calling byArchive but no archive is specified in the method signature